lib/postman/auth/awsv4.js

```diff
@@ -1,7 +1,17 @@
+const aws4  = require('aws4');
+const AWS_CREDS = {
+    accessKeyId: process.env.AWS_ACCESSKEY,
+    secretAccessKey: process.env.AWS_SECRETKEY
+};
 
 module.exports = {
-  enabled: false,
-  header: function (options) {
+  enabled: true,
+  signedHeaders: function ({service, path}) {
+    if (service && path) {
+    	const { service, path}
+    	return aws4.sign({ service, path }, AWS_CREDS)
```

Comment:
The AWS v4 `Authorization` header needs to be calculated at test script runtime, not at point of conversion/code-generation. So you'd need to generate code that imports and uses a third-party lib for handling AWS v4 auth, see https://support.loadimpact.com/4.0/test-scripting/examples/#aws-signature-v4-authentication.

This issue from a sister project is also relevant here: https://github.com/loadimpact/jmeter-to-k6/issues/3

Reply:
ok got it, thanks for the feedback